### ISSUE TITLE
Not filling in the polygon

### ISSUE THREAD

**Issue by IronReignRobotics** [waynebaswell/precision-mule-server#1]
The generated waypoints seem to not be filling the entire polygon. Attached are the polygon file and the waypoint file for a shape over a softball field in Dallas.

These were generated by using the precision mule website, not this repo directly. So I might be submitting this issue in the wrong place.

The problem also exists when defining a simple rectangular shape at the default location of precision mule.

There's also a strange artifact that I don't understand. There is a placemark labelled "Go" in the center of my view on the precision mule site. It has a dotted line leading to the north. Not sure what that is about.

Thanks for the really helpful work you've published here. I also love the RTK setup video.

[issue.zip](https://github.com/waynebaswell/precision-mule-server/files/5178208/issue.zip)

**Comment by IronReignRobotics** [comment]
Haven't found a pattern to the issue. Sometimes it happens with very simple shapes.

![rect](https://user-images.githubusercontent.com/6445257/92307417-051c6c80-ef5c-11ea-8fda-5fedc08b1152.PNG)

**Comment by IronReignRobotics** [comment]
doh, real homer moment there. Figured out the Go placemark was a hint where to start the path and it's angle maps to the specified angle. That placemark was not visible when I defined the shape and generated the initial waypoints. It showed up later on refreshing the page. It would be nice to have the Go placemark visible in the view when building the mission, but that's really a feature request for the UI code.
